UI/Components/SplitComponent.cs

```diff
@@ -673,7 +676,26 @@ protected void UpdateAll(LiveSplitState state)
                         NameLabel.Text = Split.Name;
                 }
 
-                var splitIndex = state.Run.IndexOf(Split);
+                if (Settings.AutomaticAbbreviation)
```

Comment:
Can you simplify this code so that it's like the code in LiveSplit.Splits? https://github.com/LiveSplit/LiveSplit.Splits/blob/master/UI/Components/SplitComponent.cs#L324

Inside of this `if`, you're effectively re-implementing the logic above. Instead of doing this, you can set a variable in the logic above, and then use that below.